binderhub/builder.py

```diff
@@ -215,6 +215,7 @@ def escape(s):
                 image_found = False
             else:
                 image_found = True
+            image_found = False
```

Comment:
left over from debugging?

Reply:
oh geez, yes. This would be a problem! Fixed.